pkgs/applications/misc/themix/theme-oomox/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, stdenv, fetchFromGitHub, python3, bc, sassc, glib, libxml2, gdk-pixbuf
+, gtk-engine-murrine
+, fetchpatch
+}:
+
+stdenv.mkDerivation rec {
+  pname = "themix-theme-oomox";
+  version = "1.12.r2.g0db28619";
+
+  src = fetchFromGitHub {
+    owner = "themix-project";
+    repo = "oomox-gtk-theme";
+    rev = "e0d729510791b4797e80610b73b5beea07673b10";
```

Comment:
Why are we not fetching https://github.com/themix-project/oomox-gtk-theme/releases/tag/1.12?

Reply:
It's because version 1.12.r2 has some bug fixes for gtk3.20+.